app/_data/docs_nav_kic_2.7.x.yml

```diff
@@ -148,5 +150,5 @@ items:
         url: /references/prometheus
       - text: Feature Gates
         url: /references/feature-gates
-      - text: Gateway API Support
+      - text: Supported Gateway API Features
```

Comment:
Do you really mean to only rename this page to `Supported Gateway API Features` only in docs of 2.7, but remain the original name in docs for older versions?

Reply:
I agree that older versions of the docs should also be renamed to match.